main.tf

```diff
@@ -48,3 +48,27 @@ resource "azurerm_kubernetes_cluster" "aks" {
     }
   }
 }
+
+resource "null_resource" "run_kured" {
```

Comment:
Thanks @michaelbrownlnr 
Looks like it does what we need.

I am interested to know why you've used a null_resource instead of the Terraform Providers
(I've listed a few in-line below :))

Thanks

Reply:
I agree that running infrastructure as code is more desirable than running imperative commands, but I changed how I approached the task after reading the lack of customization associated with the chart that was mentioned in the following  article and saw a local exec resource as more likely  to accomplish the task with more leeway for any additional customizations we might need later on. I'll do some testing with a helm provider in terraform:
https://alwaysupalwayson.blogspot.com/2020/01/flexible-kured-deployment-with-its-helm.html